marlowe-playground-client/src/Contrib/Type/Proxy/Generic.purs

```diff
@@ -0,0 +1,40 @@
+module Contrib.Type.Proxy.Generic where
```

Comment:
The current convention in our codebase is something like

```
module Type.Proxy.Extra
```

Though again, I could be convinced to switch to `Contrib.`

Reply:
I've just missed `Extra` - I can turn this into `Extra` to keep consistency in the codebase.

It is probably a matter of a habit but maybe it is a bit easier to organize namespaces (split extras) when `Contrib` namespace is on the top.

I don't have a really strong preference in this regards.

Reply:
I'm leaving `Contrib` namespace for now - we can discuss pros and cons in the near future.